plone/app/contenttypes/browser/templates/folder_listing.pt

```diff
@@ -41,7 +41,7 @@
                         isAnon portal_state/anonymous;
                         normalizeString nocall: context/plone_utils/normalizeString;
                         toLocalizedTime nocall: plone_view/toLocalizedTime;
-                        show_about python:not isAnon or site_properties.allowAnonymousViewAbout;
+                        show_about python:not isAnon or context.portal_registry['plone.allow_anon_views_about'];
```

Comment:
we should not use acquisition to acces tools. Instead add a method 'registry' to plone.app.layout.globals.tools.Tools